base/BUILD.gn

```diff
@@ -1725,9 +1725,15 @@ component("base") {
     # partition_alloc code if use_partition_alloc = false because no code uses
     # partition_alloc.
     public_deps += [
-      "allocator/partition_allocator:partition_alloc",
       "allocator/partition_allocator:raw_ptr",
     ]
+
+    # TODO(cobalt, b/382115610): Re-enable use_partition_alloc
+    if (!is_cobalt) {
```

Comment:
Use `if (is_cobalt) { public_deps -= ... }` instead

Reply:
ahh the positive conditional, gotcha

Reply:
done , i try to incorporate this by default in future cl's